HIP/hip-583.md

```diff
@@ -2,221 +2,268 @@
 
-The size of the bytes in each case depends on the governing account key type. For an `ECDSA` based account the alias will always be a valid Ethereum address and will be 20 bytes long. In the `ED25519` case the alias will be a valid protobuf serialized public key of 35 or 68 bytes depending on encoding.
+The size of the bytes in each case depends on the governing account key type. For an `ECDSA` based account the alias
+will always be a valid Ethereum address and will be 20 bytes long. In the `ED25519` case the alias will be a valid
+protobuf serialized public key of 35 or 68 bytes depending on encoding.
 
 ## Security Implications
 
-The `lazy-create` account flow requires additional checks be added to ensure transactions involving an account with only a `public-address` are not processed without a matching `public-key`. 
+The `lazy-create` account flow requires additional checks be added to ensure transactions involving an account with only
+an `alias` and no key (i.e. Hollow accounts) are not processed. 
 
 ## How to Teach This
 
-- Additional documentation on protobufs
+- Additional documentation on protobufs
 - SDK examples should be written to highlight
-    - `CryptoCreate` alias setting using a `public-key`
-    - `CryptoCreate` alias setting using a `public-address` i.e. lazy account creation flow.
-    - `CryptoTransfer` alias setting using a `public-address`
+    - `CryptoCreate` alias setting using an `evm address alias`
+    - `CryptoCreate` alias setting using a `key alias`
+    - `CryptoTransfer` alias setting using an `evm address alias`
 - Doc site tutorials utilizing SDKs should be written to highlight alias setting and the lazy account creation flow.
 
 ## Reference Implementation
 
-The reference implementation must be complete before any HIP is given the status of “Final”. The final implementation must include test code and documentation.
+The reference implementation must be complete before any HIP is given the status of “Final”. The final implementation
+must include test code and documentation.
 
 ## Rejected Ideas
 
-- A special reserved `ECDSA` based `aliasKey` on all accounts. This would have ensured only a single key was used on contract interactions.
-- A special reserved `ECDSA` based `aliasKey` and an additional `evmAlias`. This would have provided complete separation of all EVM interactions with Hedera account keys.
-
-In a way, this section can be thought of as a breakout section of the Rationale section that focuses specifically on why certain ideas were not ultimately pursued.
+- A special reserved `ECDSA` based `aliasKey` on all accounts. This would have ensured only a single key was used on
+  contract interactions.
+- A special reserved `ECDSA` based `aliasKey` and an additional `evmAlias`. This would have provided complete separation
+  of all EVM interactions with Hedera account keys.
+- The ability to call `CryptoCreate` with just the `evm address alias` and not the public key. Use a `CryptoTransfer`
```

Comment:
Disagree here.
This was an explicit request by wallets to support their current flows.
Creating 0 balance accounts is important to them.
Additionally, we can assume developers would have access to the public key used to derive the evm address, but we can't presume all business practices will have the public key in the same place as the evm address.
Many web3 flows might preserve a list of evm address in one places and the private keys in another. They rarely deal with public keys as Hedera does

Reply:
I do not understand.

If the account has zero balance, why would I want to pay for the account's creation? If the account is not going to have zero balance, then I will need to do a crypto transfer to create the account, in which case it will be auto-created if I do the transfer first. If I have custody of the key, then I can create a normal account. If I do not have custody of the key, and I do no intend to transfer HBAR into the account, why would I pay to create the account?

Reply:
I think there's a potential here, but it remains a hypothesis and for that reason it's best to not add.
I'm okay with this as a rejected idea.
if we get the data we can add it when propose it again when the time is right